kyo-tag/shared/src/main/scala/kyo/tags.scala

```diff
@@ -19,7 +19,7 @@ object Tag:
     inline given apply[T]: Tag[T] = ${ tagImpl[T] }
 
     extension [T](t1: Tag[T])
-        def erased: Tag[Any] = t1.asInstanceOf[Tag[Any]]
+        inline def erased: Tag[Any] = t1.asInstanceOf[Tag[Any]]
```

Comment:
This method not being inlined was preventing optimizations when suspending `Abort` and `Env` with erased tags.